modules/meeting/app/forms/meeting/time_group.rb

```diff
@@ -64,7 +64,7 @@ class Meeting::TimeGroup < ApplicationForm
         label: Meeting.human_attribute_name(:duration),
         visually_hide_label: false,
         required: true,
-        caption: I18n.t("text_in_hours"),
+        trailing_visual: { text: { text: I18n.t("datetime.units.hour.other") } },
```

Comment:
I see two issues here: 

<img width="182" alt="Bildschirmfoto 2025-01-21 um 10 28 11" src="https://github.com/user-attachments/assets/c0bcedf8-3bf4-4faf-a856-5d8b74c068d7" />

* The unit is shown twice (one time after the value, and one time at the end of the field)
* The unit at the end is truncated. 

So I'd suggest, to get rid of the unit after the value, and at the end use an abbreviated form like "h". We may have to ask product about that however

Reply:
👍🏻 for "h"

Reply:
Hi @myabc as discussed [here](https://community.openproject.org/projects/stream-planning-and-reporting/work_packages/59860/activity#activity-16), We continue with 'in hours' as caption for the input field.